### ISSUE TITLE
I believe this is Python 2 code and can be safely removed. correct?

### ISSUE THREAD

**Issue by jvanasco** [sqlalchemy/dogpile.cache#243]
https://github.com/sqlalchemy/dogpile.cache/blob/fa090bef0efd4efc350bc85f5b7f3c085c812384/dogpile/cache/backends/redis.py#L9

i will generate the PR if someone can confirm

**Comment by zzzeek** [comment]
yes that is python 2 only, absolute_imports are standard in python 3

**Comment by jvanasco** [comment]
That's what I thought, but it was weird seeing it so I second-guessed myself.

PR is in #244
